tests/e2e/tests/globals.js

```diff
@@ -0,0 +1,44 @@
+import { Page } from '@playwright/test';
+import { LizmapConnectionPage } from './pom/lizmap-connection-page'
+import path from 'path';
+
+/**
+ * Performs the authentication steps
+ * @param {Page} page The page object
+ * @param {string} login The login
+ * @param {string} password The password
+ * @param {string} user_file The path to the file where the cookies will be stored
+ */
+export async function auth_using_login(page, login, password, user_file) {
```

Comment:
```suggestion
export async function authUsingLogin(page, login, password, user_file) {
```

It seems in JS that function are lower camelcase

Reply:
Fixed in [58507f0](https://github.com/3liz/lizmap-mapbuilder-module/pull/69/commits/58507f086bb52752f09beebb379f22ac6ffd3a4b).